ESMA/APITesting.py

```diff
@@ -0,0 +1,67 @@
+import unittest
+import main
+import os
+import pickle
+from sympy import E as e
```

Comment:
I've changed it, but this is how it was in the original version (the one in master at the moment). So are you sure this will be more stable? I haven't fiddled with it enough to have an opinion

Reply:
well that was Yoav's code :) 
better to use sympy.e